src/org/opendatakit/briefcase/export/ExportForms.java

```diff
@@ -133,10 +135,10 @@ public void updateDefaultConfiguration(ExportConfiguration configuration) {
     defaultConfiguration = configuration;
   }
 
-  public ExportConfiguration getConfiguration(String formId) {
-    return Optional.ofNullable(customConfigurations.get(formId))
+  public ExportConfiguration getConfiguration(FormStatus form) {
+    return Optional.ofNullable(customConfigurations.get(form.getFormDefinition().getFormId()))
```

Comment:
Duplicated code. It should be `getFormId(form)`. Also, if/when you change `ExportConfiguration.fallingBackTo()`, the signature change on `ExportForms.getConfiguration()` should be rolled back...